pkg/csi_driver/node.go

```diff
@@ -450,3 +476,114 @@ func getFSStat(path string) (available, capacity, used, inodesFree, inodes, inod
 	inodesUsed = inodes - inodesFree
 	return
 }
+
+// nodeStageVolumeUpdateLockInfo updates lock info after NodeStageVolume succeed.
+func (s *nodeServer) nodeStageVolumeUpdateLockInfo(ctx context.Context, req *csi.NodeStageVolumeRequest) error {
+	volumeID := req.GetVolumeId()
+	// No-op if driver does not support lock release.
+	if s.kubeClient == nil {
+		klog.Infof("Skip lock release info update for volume %s: kubeClient not initialized for node driver", volumeID)
+		return nil
+	}
+	// No-op if filestore instance not support lock release.
+	attr := req.GetVolumeContext()
+	if val, ok := attr[attrSupportLockRelease]; !ok || strings.ToLower(val) != "true" {
+		klog.Infof("Lock release is not support on volume %s: missing volume attribute %s. The volume is not dynamically provisioned or the filestore instance is not in enterprise tier", volumeID, attrSupportLockRelease)
+		return nil
+	}
+
+	// Update the configMap after successful nfs mount operation.
+	klog.Infof("NodeStageVolume storing lock info for volume %s", volumeID)
+	nodeName := s.driver.config.NodeName
+	configmapName := util.ConfigMapNamePrefix + nodeName
+	klog.Infof("NodeStageVolume getting configmap %s/%s for volume %s", util.ConfigMapNamespace, configmapName, volumeID)
+	cm, err := util.GetConfigMap(ctx, configmapName, util.ConfigMapNamespace, s.kubeClient)
+	if err != nil {
+		return err
```

Comment:
log a klog.Errorf() for easy trace in csi driver

Reply:
Done